core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -3348,19 +3348,18 @@ public void getFeatureVariableValueReturnsDefaultValueWhenFeatureIsAttachedToOne
 
     /**
      * Verify {@link Optimizely#getFeatureVariableValueForType(String, String, String, Map, FeatureVariable.VariableType)}
-     * returns the variable value of the variation the user is bucketed into
-     * if the variation is not null and the variable has a usage within the variation.
+     * returns the default variable value
+     * if the variation is not null featureEnabled is false.
```

Comment:
nit. The docstring looks ill formatted. Please fix this.

Also, missing and. `...variation is not null and featureEnabled is false.`